src/composables/useTour.ts

```diff
@@ -0,0 +1,393 @@
+    [x in T]: TourStep;
+};
+
+// TODO Remove me
+export function useFakeTx(): Transaction {
+    return {
+        transactionHash: '0x123',
+        format: 'nim',
+        timestamp: 1532739000,
+        sender: useAddressStore().activeAddress.value || '',
+        recipient: '0x123',
+        senderType: 'nim',
+        recipientType: 'nim',
+        blockHeight: 1,
+        blockHash: '0x123456789ABCDEF',
+        value: 100_000,
+        fee: 1,
+        feePerByte: 1,
+        validityStartHeight: 1,
+        network: 'testnet',
+        flags: 0,
+        data: {
+            raw: 'My awesome data',
+        },
+        proof: {
+            raw: 'ES256K',
+        },
+        size: 1,
+        valid: true,
+        state: 'confirmed',
+    };
+}
+function getOnboardingTourSteps({ root }: SetupContext): TourSteps<MobileOnboardingTourStep> {
+    const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+    const closeAccountOptionsModal = async () => {
+        const sidebar = (document.querySelector('.column-sidebar') as HTMLDivElement);
+        sidebar!.removeAttribute('data-non-interactable');
+        await root.$nextTick();
+
+        const closeBtn = (document.querySelector('.modal .close-button') as HTMLDivElement);
+        closeBtn.click();
+
+        await sleep(500); // TODO Check this random value
+    };
+
+    const openAccountOptionsModal = async () => {
+        const account = document.querySelector('.sidebar .account-menu') as HTMLButtonElement;
+        account.click();
+        await sleep(500); // TODO Check this random value
+    };
+
+    const steps: TourSteps<MobileOnboardingTourStep> = {
+        [MobileOnboardingTourStep.FIRST_ADDRESS]: {
+            path: '/',
+            tooltip: {
+                target: '.address-list > .address-button .identicon img',
+                content: [
+                    'This is your first address, represented by your avatar.',
+                    'Tap it to open the address details.',
+                ],
+                params: {
+                    placement: 'bottom-start',
+                },
+            },
+            lifecycle: {
+                prepareDOMNextPage: async () => {
+                    if (root.$route.path === '/') {
+                        const addressButton = document
+                            .querySelector('.address-list > .address-button') as HTMLButtonElement;
+                        addressButton.click();
+                    }
+                },
+                onMountedStep: (cbu: () => void) => {
+                    const addressButton = document.querySelector('.address-list > .address-button');
+
+                    // eslint-disable-next-line no-unused-expressions
+                    addressButton?.addEventListener('click', () => cbu(), { once: true });
+                    // TODO Remove listener somehow if user clicks on the '>' button instead on the address item
```

Comment:
A method I learned from Svelte was that the `onMounted` function can _return_ a callback that is run when the component _unmounts_. That could work here too, as long as that callback (if given) is guaranteed to be run.